script/tool/lib/src/common/repository_package.dart

```diff
@@ -58,6 +58,15 @@ class RepositoryPackage {
   bool get isPlatformInterface =>
       directory.basename.endsWith('_platform_interface');
 
+  /// True if this appears to be a platform implementation package, according to
+  /// repository conventions.
+  bool get isPlatformImplementation =>
```

Comment:
Should this be determined by the presence of the `implements` pubspec.yaml property?

Reply:
If we ever find that this heuristic isn't working, that's the way to go (and by having it centralized here was can easily improve it later). That's way more involved though since it requires loading and parsing the pubspec, and I suspect the heuristic may well be all we ever need.